src/Image/Image.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { props, t } from '../utils';
+import getUrl from './getUrl';
+
+export type ImageProps = {
+  /** path to the image */
+  src: string,
+  /** used to resize the image to this width before downloading it */
+  width?: number,
+  /** used to resize the image to this height before downloading it */
+  height?: number,
+  /** can be a number from 1 to 100 or "auto". Use "auto" to let cloudinary decide the quality for you */
+  quality?: number | 'auto',
+  /** which format the image should be. Use "auto" to let cloudinary decide the format for you */
+  format?: 'jpeg' | 'png' | 'wdp' | 'gif' | 'auto',
+} & React.HTMLAttributes<HTMLImageElement>
+
+export const Props = {
+  src: t.String,
+  width: t.maybe(t.Number),
+  height: t.maybe(t.Number),
+  quality: t.maybe(t.union([t.Number, t.enums.of(['auto'])])),
+  format: t.maybe(t.enums.of(['jpeg', 'png', 'wdp', 'gif', 'auto']))
+};
+
+/** A replacement for `<img>` to serve optimized images through a CDN in production */
+@props(Props, { strict: false })
+export default class Image extends React.Component<ImageProps> {
```

Comment:
whoops, missed this. 
This should be a `PureComponent`

Reply:
But it wasn't before :(

Reply:
ok, leave it as is, even if it shouldn't hurt